src/utils/keyboardEvents.jsx

```diff
@@ -0,0 +1,37 @@
+const ACTION_KEYS = [
+  'ARROWLEFT',
+  'ARROWRIGHT',
+  'SPACE',
+];
```

Comment:
Can you add for arrow up and arrow down to increase and decrease the volume respectively?

Reply:
Just saw that that was not integrated